examples/practice-typescript/src/components/Button/index.tsx

```diff
@@ -0,0 +1,12 @@
+import './index.scss';
+
+interface ButtonProps {
+  text: string;
+  style: 'btn__action' | 'btn__add';
```

Comment:
This is not style, it is a class naming for style. Pls change other name like `styleClass`

Reply:
Fixed